pkgs/development/python-modules/sfepy/default.nix

```diff
@@ -15,23 +21,34 @@
 , openssh
 , pyvista
 , pytest
-, pythonOlder
+, stdenv
 }:
 
 buildPythonPackage rec {
   pname = "sfepy";
-  version = "2023.1";
-  format = "setuptools";
+  version = "2024.1";
+  pyproject = true;
+
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "sfepy";
     repo = "sfepy";
     rev = "release_${version}";
-    hash = "sha256-PuU6DL9zftHltpYI9VZQzKGIP8l9UUU8GVChrHtpNM0=";
+    hash = "sha256-r2Qx9uJmVS4ugJxrIxg2UscnYu1Qr4hEkcz66NyWGmA=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [
+    cmake
+    ninja
+    oldest-supported-numpy
+    setuptools
+    scikit-build
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  dependencies = [
```

Comment:
Let's clean up the dependencies.

`cython_0` should go to `build-system`, and `mpi4py`, `psutil`, and `openssh` should be removed.
https://github.com/sfepy/sfepy/blob/be30a1f400c07a756795eef1607bdd353e524a28/setup.py#L199-L209

Reply:
It still builds fine.